M/ModularForms/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+using BinaryBuilder, Pkg
+import Pkg.Types: VersionSpec
+
+name = "ModularForms"
+version = v"0.1.0"
+
+sources = [GitSource("https://gitlab.com/mraum/ModularForms_jll_src.git",
+                     "4d81d41e625fde3e36dc86df5061313a8447cacd")]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ModularForms_jll_src
+if [[ ${target} == *musl* ]]; then
+   export CFLAGS=-D_GNU_SOURCE
+fi
+./configure --prefix=$prefix --disable-static --enable-shared --with-gmp=$prefix --with-mpfr=$prefix --with-flint=$prefix --with-arb=$prefix CC=gcc ${extraflags}
```

Comment:
Why forcing `gcc` as compiler?  Also, there is no `extraflags` variable

Reply:
These two are mistakes, that I'll fix. Thank you!